resources/contenttweaker/lang/en_us.lang

```diff
@@ -253,27 +255,88 @@ e2ee.tconstruct.material.researcher.name=Researcher
 e2ee.tconstruct.material.researcher.description=Based on your research in thaumonomicon, there are various effects!
 e2ee.tconstruct.material.first_stand.name=First Stand
 e2ee.tconstruct.material.first_stand.description=Consume experience before taking damage.
+e2ee.tconstruct.material.robust.name=Robust
+e2ee.tconstruct.material.robust.description=Increase health
 
 warp.sword.warning=§o§5 You are too sane to use that power!§r
 
-warp.sword.speak.random.0=§o§5I will put as many as I can in the grave, just one swing is enough!§r
-warp.sword.speak.random.1=§o§5Rivers of blood will turn into seas...§r
-warp.sword.speak.random.2=§o§5Why don't we just kill all livings?...§r
-warp.sword.speak.random.3=§o§5The J'zahar calls for prey!§r
-warp.sword.speak.random.4=§o§5The more we kill, the more thirsty we become
-warp.sword.speak.random.5=§o§5The sky darkens, the crying stops, life ends...
-warp.sword.speak.random.6=§o§5Await for Crimson Dawn and tears of terror
-warp.sword.speak.random.7=§o§5Dread... Hate... Despair...
-warp.sword.speak.random.8=§o§5The Gatekeeper is always watching...
+warp.sword.speak.random.0=§o§5Quick reminder: I'm the master, you're my slave.§r
```

Comment:
Why all speak messages now have `§r` at the end of each line? Is it necessary?

Reply:
That was syntax to reset any formatted text, but since every line is displayed separately, I think I can remove it